packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -355,58 +355,55 @@ export class VaModal {
 
     return (
       <Host
-      aria-label={ariaLabel}
-      aria-modal="true"
-      role={ariaRole(status)}
+        aria-label={ariaLabel}
+        aria-modal="true"
+        role={ariaRole(status)}
+        visible={visible}
```

Comment:
@bkjohnson This change `visible={visible}` reflects in the DOM allowing the styles to be applied properly when using va-modal inside other web components.

Reply:
Could we have the `visible` prop have `{reflect: true}` and get the same result?

Reply:
You're correct. Updating to that.